README.md

```diff
@@ -35,6 +35,15 @@ For an image with all additional components (like NodeRed, logi-rts and Teamview
 
 `customize_image.sh <raspbian-image>`
 
+
+### Install debian packages into Revolution Pi Image
+
+To build an image with only update of local packages and Revolution Pi packages, you can get it by following command with a released revolution pi image:
+
+`install_debs_into_image.sh <revpi-image>`
+
+The script requires the same running environment as previous customizing image
```

Comment:
Why? Shouldn't it be enough to have all relevant tools installed and run the script against eg. the buster-latest image?

Reply:
I mean the same environment as this:
"The script requires root privileges, an armhf system (RasPi or VM) and internet connectivity.

If no armhf system is available, a crossbuild can be done with the qemu user static tools:

```
sudo apt-get install qemu-user-static binfmt-support
```"

Reply:
OK, then I'd opt to remove this line. From the textual flow it should be clear, that the requirements are mandatory for booth scripts.